packages/mdc-dialog/mdc-dialog.scss

```diff
@@ -208,6 +208,25 @@
 
 // postcss-bem-linter: end
 
+.mdc-dialog__title + .mdc-dialog__content {
```

Comment:
Move these two style rules next to .mdc-dialog__content above?

Reply:
I'll try that, but I suspect I'll have to exclude them from bem-linter anyway and might also get increasing-specificity warnings. That's why I put them at the end.

Reply:
Yep, had to exclude from bem-linter. No other problems though. Also apparently had a HTML lint error that I fixed.